governance.md

```diff
@@ -79,14 +79,65 @@ process.
 
 ## Elections
 
-The election process is not yet established. It MUST be done by the next election cycle. It MUST
-define, at least, the following:
-
-1. The actions required to fulfill a change in the committee composition. For example, when a new
-   committee member joins, which groups should they be added to, and, conversely, when a member
-   leaves, which groups should they be removed from.
-2. The eligibility requirements of voters and nominees.
-3. The voting process.
+> The elections process may become more strict over time, depending on the evolving
+nature of the community.
+
+### Voter Eligibility
+
+The elections are open for anyone to vote. The community asks that voters:
+
+1. Vote only if they are active contributors to the community.
+2. Vote only once.
+
+### Candidate Eligibility
+
+Candidates must be active contributors to the community, and must be nominated by an
+eligible voter.
+
+### Nominations Process
+
+Nominations should be sent to konflux@googlegroups.com. The email should contain: email
+address, github handle, company affiliation, and Konflux-CI project(s) they contribute
+to, for both the nominee and the nominating contributors.
+
+One CAN nominate themselves.
+
+Any nominee who accepts the nomination will be on the ballot.
+
+### Election Process
+
+Elections will be held using time-limited
+[Condorcet](https://en.wikipedia.org/wiki/Condorcet_method) ranking on
+[CIVS](https://civs.cs.cornell.edu/). The top vote getters will be elected to the open
+seats.
+
+In case the number of seats is greater or equal to the number of candidates,
+voters will also be able to vote against each candidate being elected, and a candidate
+will only be elected if more votes are casted for the candidate than against.
+
+Details about the schedule and logistics of the election will be announced in a timely
+manner by the election officers to eligible candidates and voters via the
+konflux@googlegroups.com mailing list.
+
+### Election Officers
+
+For every election, the governance committee will choose election officers as they deem
+appropriate and sufficient (e.g. headcount and identity).
+
+### Changes to Governing Committee
```

Comment:
@ralphbean @arewm @brianwcook can you suggest some content for this section, according to how you currently run things.

Reply:
I don't think it currently exists, but let's create a "konflux-governance-committee" team in github and then the list can maybe be shortened to just add and remove from that team. We can apply the team to different situations as they arise.

The one thing I have in mind right now is the CODEOWNERS file for https://github.com/konflux-ci/community.

Reply:
ok. I [created it](https://github.com/orgs/konflux-ci/teams/konflux-governance-committee), but I cannot set you as maintainers and I cannot leave (probably unless there's another maintainer). Can you try setting yourself as a maintainer and see if you can make me leave :)

Reply:
> The one thing I have in mind right now is the CODEOWNERS file for https://github.com/konflux-ci/community.

I suppose that once we have this group properly set, we can have the CODEOWNERS file reference that group instead of your names. Right?

Reply:
SLSA recently re-wrote its governance document. It has some content on term rotation if it helps: 
https://github.com/slsa-framework/governance/blob/main/5._Governance.md

Reply:
After reading into this link, I changed contributors to participants.